src/pages/TakeActionTips.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+    <DefaultLayout>
+      <div class="layout-constrained">
+        <section class="takeaways">
+        <h2>What Should We Do About This?</h2>
```

Comment:
Can you switch this to an `<h1>` and add `id="main-content" and tabindex="-1"`? That matches other pages and will make it the "Skip to Main Content" link target for keyboard users.

Reply:
Fixed

Reply:
@brayckner - I think you fixed it in the wrong place, this line of code hasn't changed! The takeaways heading was correct as an `h2`. I might just fix it real quick so we can merge this in, but wanted to mention

Reply:
I pushed up a correction to match the headings on the other pages. @vkoves